zingolib/src/lightclient.rs

```diff
@@ -2055,6 +2056,14 @@ impl LightClient {
 
         Ok(())
     }
+
+    pub fn get_wallet_file_location(&self) -> PathBuf {
```

Comment:
Can you add  a doc-comment to this fn and start a CHANGELOG.md following the style here:

https://github.com/zcash/librustzcash/blob/main/zcash_primitives/CHANGELOG.md